README.org

```diff
@@ -44,12 +44,17 @@
   #+BEGIN_SRC lua
   world:draw()
   #+END_SRC 
+
 *** can be repurposed to draw in-game shapes/sprites!
   Just redefine :draw on your collider objects, 
   add other objects with :draw methods to world.colliders.
   # TODO rename world.colliders to world.to_draw?
   Call world:draw(<alpha>, true) to draw physics boundaries in addition to
   self-defined :draw methods
+  
+  You can ensure some colliders are drawn over others with `Collider:setDrawOrder(number)`.
+  Colliders with higher draw orders will be drawn over those with smaller draw orders.
+  The default is 0, and the draw order can be negative.
```

Comment:
Also, I think it would make more sense to name it "draw priority", since colliders with higher values take precedence over smaller ones, and not vise-versa.